### PR TITLE
Fix many py2-style prints in docs and commented code.

### PR DESCRIPTION
**Pull request by anntzer**
The main interest was to transform py2-style print statements that still appeared in some docstrings (e.g. for %history) and would thus show up when looking at the docs either interactively or in the rendered website.  I also fixed a number of commented-out print statements at the same time, mostly because it was easier for me to just regexp-change all of them so that I could focus on the others.

### CODE REVIEW

File: IPython/core/tests/tclass.py [hunk 1]
```diff
@@ -26,7 +26,7 @@ def __del__(self):
     if name.startswith('C'):
         c = C(name)
 
-#print >> sys.stderr, "ARGV:", sys.argv  # dbg
+# print(>> sys.stderr, "ARGV:", sys.argv)  # dbg
```
Comment (Carreau) [inline IPython/core/tests/tclass.py:29]:
```suggestion
# print("ARGV:", sys.argv, file=sys.stderr)  # dbg
```

File: IPython/testing/plugin/ipdoctest.py [hunk 2]
```diff
@@ -68,7 +68,7 @@ def check_output(self, want, got, optionflags):
         ret = doctest.OutputChecker.check_output(self, want, got,
                                                  optionflags)
         if not ret and self.random_re.search(want):
-            #print >> sys.stderr, 'RANDOM OK:',want  # dbg
+            # print(>> sys.stderr, 'RANDOM OK:',want)  # dbg
```
Comment (Carreau) [inline IPython/testing/plugin/ipdoctest.py:71]:
```suggestion
            # print('RANDOM OK:',want, file=sys.stderr)  # dbg
```

**Review by Carreau — COMMENTED** [review]
2 minor things I'll apply directly and merge.

File: IPython/testing/plugin/ipdoctest.py [hunk 3]
```diff
@@ -68,7 +68,7 @@ def check_output(self, want, got, optionflags):
         ret = doctest.OutputChecker.check_output(self, want, got,
                                                  optionflags)
         if not ret and self.random_re.search(want):
-            #print >> sys.stderr, 'RANDOM OK:',want  # dbg
+            # print('RANDOM OK:',want, file=sys.stderr)  # dbg
```
Comment (westurner) [inline IPython/testing/plugin/ipdoctest.py:71]: So old! :+1:

**Pull request merged**